### DISCUSSION TITLE
No Display Capture option - Ubuntu 22

### DISCUSSION THREAD

**Discussion by hermans** [obsproject/obs-studio#7002] [General]
Hello, 

I just installed OBS on Ubuntu 22, i installed use snap, i don't see display capture (share screen)
![Screenshot from 2022-08-09 11-20-59](https://user-images.githubusercontent.com/938767/183563430-a8aa761d-6350-4797-bb5c-a9b602f78ba5.png)

**Comment by RytoEX** [comment]
We do not maintain the Snap package. Per [their description](https://snapcraft.io/obs-studio):
> This is a community-supported modified build of OBS Studio; please file issues on the Snapcrafters GitHub: https://github.com/snapcrafters/obs-studio/issues

We support Ubuntu via Flatpack and our PPA. Please see the [Install Instructions wiki page](https://github.com/obsproject/obs-studio/wiki/install-instructions#supported-builds) for more information.

**Reply by hermans** [reply]
I just remove from snap, and installed use PPA, but it have same result, but the option is less :
![Screenshot from 2022-08-09 11-53-36](https://user-images.githubusercontent.com/938767/183567368-688da1eb-f05a-4878-865c-4d1d9b340391.png)

**Reply by RytoEX** [reply]
For support, please seek assistance in the [Linux Support](https://obsproject.com/forum/list/linux-support.34/) section of our forum, or the #linux-support channel on [our Discord](https://obsproject.com/discord).

**Reply by hermans** [reply]
Ok, it'be been asked by someone else (but it still not fix the problem)
> https://obsproject.com/forum/threads/obs-no-display-screen-capture-option.156314/

**Reply by hermans** [reply]
fixed: https://askubuntu.com/questions/1347489/ubuntu-22-04-any-screen-recorders-not-working-showing-black-screen-only

**Reply by jordimontana82** [reply]
It's because Ubuntu22.04 uses Wayland by default.  I didn't have to install anything special, just on the sign in screen, pick the settings (down right) and pick Ubuntu on XOrg. Similar to this answer https://askubuntu.com/a/1347522
